test/formulas/reference/testcase.js

```diff
@@ -111,6 +111,7 @@ module.exports = {
         'ROWS("A1")': FormulaError.VALUE
     },
 
+
```

Comment:
Where's the test cases?

Reply:
huh, weird this is different on my end, Ill repull from master